akka-remote/src/test/scala/akka/remote/serialization/MiscMessageSerializerSpec.scala

```diff
@@ -25,7 +26,9 @@ object MiscMessageSerializerSpec {
     akka.actor.serialization-bindings {
       "akka.remote.serialization.MiscMessageSerializerSpec$TestException" = akka-misc
       # not enabled by default
-      "akka.Done" = akka-misc
+      "akka.Done"                = akka-misc
+      "akka.actor.Address"       = akka-misc
+      "akka.remote.UniqueAddress" = akka-misc
     }
```

Comment:
I was wondering... if we say `allow-java-serialization = off` should we not enable all such serializers then?

Reply:
it adds some special casing, but we already have some of that for `allow-java-serialization` so that could be fine, and convenient

Reply:
Done; I can't see why we did not add that Done as well as those types to `additional-serialization-bindings` directly. I mean, if one enables them things are made incompatible between a node which enabled them and a node which did not anyway, so AFAICS we can just include those bindings in there?

Reply:
Aaaa, my bad, those are ENABLED by default. Adding a new section that is enabled only when java serialization is disabled.